scipy/optimize/tests/test__numdiff.py

```diff
@@ -576,19 +616,22 @@ def test_all(self):
         np.random.shuffle(order)
         groups_2 = group_columns(A, order)
 
-        for method, groups, l, u in product(
-                ['2-point', '3-point', 'cs'], [groups_1, groups_2],
-                [-np.inf, self.lb], [np.inf, self.ub]):
-            J = approx_derivative(self.fun, self.x0, method=method,
-                                  bounds=(l, u), sparsity=(A, groups))
-            assert_(isinstance(J, csr_array))
-            assert_allclose(J.toarray(), self.J_true, rtol=1e-6)
-
-            rel_step = np.full_like(self.x0, 1e-8)
-            rel_step[::2] *= -1
-            J = approx_derivative(self.fun, self.x0, method=method,
-                                  rel_step=rel_step, sparsity=(A, groups))
-            assert_allclose(J.toarray(), self.J_true, rtol=1e-5)
+        with MapWrapper(2) as mapper:
+            for method, groups, l, u, mf in product(
```

Comment:
checks that `approx_derivative` still works for sparse derivatives if using the `workers` kwd.